xml/ha_storage_basics.xml

```diff
@@ -0,0 +1,134 @@
+  </dm:docmanager>
+ </info>
+ <sect1 xml:id="sec.ha.storage.dlm.protocol">
+  <title>Protocols for DLM Communication</title>
+
+  <para>
+   DLM can either use the TCP or SCTP protocol for communication. If network
+   device bonding cannot be used for any reason, the second best choice is to
+   define a redundant communication channel (a second ring) in &corosync;. If
+   you have configured redundant communication rings in &corosync.conf;, DLM
+   will automatically SCTP. However, if you want DLM to use TCP for
+   communication, disable <guimenu>rrp_mode</guimenu> by setting it to
+   <literal>none</literal>. For details on configuring a redundant
+   communication channel in &corosync; and setting <guimenu>rrp_mode</guimenu>,
+   see <xref linkend="pro.ha.installation.setup.channel2"/>.
+  </para>
+ </sect1>
+ <sect1 xml:id="sec.ha.storage.generic.dlm.config">
+  <title>Configuring DLM Cluster Resources</title>
+
+  <para>
+   DLM uses the cluster membership services from Pacemaker which run in user
+   space. Therefore, DLM needs to be configured as clone resource that is
+   present on each node in the cluster.
+  </para>
+
+  <note>
+   <title>DLM Resource for Several Solutions</title>
+   <para>
+    As &ocfs;, GFS2, cluster MD, and cluster LVM (lvmlockd) all use DLM, it is
+    enough to configure one resource for DLM. As the DLM resource runs on all
+    nodes in the cluster it is configured as a clone resource.
+   </para>
+   <para>
+    If you have a setup that includes both &ocfs; and LVM2, configuring
+    <emphasis>one</emphasis> DLM resource for both &ocfs; and LVM2 is enough.
+   </para>
+  </note>
+
+  <procedure xml:id="pro.dlm.resources">
+   <title>Configuring a Base Group for DLM</title>
+   <para>
+    The configuration consists of a base group that includes several primitives
+    and a base clone. Both base group and base clone can be used in various
+    scenarios afterward (for both &ocfs; and LVM2, for example). You only need
+    to extended the base group with the respective primitives as needed. As the
+    base group has internal colocation and ordering, this simplifies the
+    overall setup as you do not need to specify several individual groups,
+    clones and their dependencies.
+   </para>
+   <para>
+    Follow the steps below on one node in the cluster:
+   </para>
+   <step>
+    <para>
+     Start a shell and log in as &rootuser; or equivalent.
+    </para>
+   </step>
+   <step>
+    <para>
+     Run <command>crm</command> <option>configure</option>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Enter the following to create the primitive resource for DLM:
+    </para>
+<screen>&prompt.crm.conf;<command>primitive</command> dlm ocf:pacemaker:controld \
+  op monitor interval="60" timeout="60"</screen>
+   </step>
+   <step>
+    <para>
+     Create a base-group for the DLM resource:
+    </para>
+<screen>&prompt.crm.conf;<command>group</command> base-group dlm</screen>
```

Comment:
Did we define groups with the `g-` prefix?

Reply:
Yep, thanks for the reminder: we use 'g-' for groups, 'cl-' for clones. As the procedure still stems from the time _before_ we defined our naming conventions, it did not contain the prefixes yet. Fixed now.